<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RHBZ</class>
 <widget class="QMainWindow" name="RHBZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RHBZ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Рассчёт</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="Form_logic">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Вместимость парка</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Park_capacity">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="text">
        <string>200</string>
       </property>
       <property name="placeholderText">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Стоимость покупки РХМ-6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Price_buy_0">
       <property name="text">
        <string>10</string>
       </property>
       <property name="placeholderText">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Стоимость ремонта РХМ-6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Price_rep_0">
       <property name="text">
        <string>1.7</string>
       </property>
       <property name="placeholderText">
        <string>1.7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Стоимость покупки РХМ-8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Price_buy_1">
       <property name="text">
        <string>15</string>
       </property>
       <property name="placeholderText">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Стоимость ремонта РХМ-8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Price_rep_1">
       <property name="text">
        <string>1.7</string>
       </property>
       <property name="placeholderText">
        <string>1.7</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Бюджет на год</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="Money_year">
       <property name="text">
        <string>1000</string>
       </property>
       <property name="placeholderText">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Колебания бюджета в новом году в %</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="Money_year_vol">
       <property name="text">
        <string>5</string>
       </property>
       <property name="placeholderText">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Возраст устаревания техники</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="Obsolete_age">
       <property name="text">
        <string>10</string>
       </property>
       <property name="placeholderText">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>261</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="Form_alg">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Размер популяции</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Кол-во поколений</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Max_generation">
       <property name="text">
        <string>300</string>
       </property>
       <property name="placeholderText">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="P_mutate">
       <property name="text">
        <string>0.2</string>
       </property>
       <property name="placeholderText">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>вер. мутации индивида</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Population_size">
       <property name="text">
        <string>800</string>
       </property>
       <property name="placeholderText">
        <string>800</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>вер. скрещивания индивидов</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="P_crossover">
       <property name="text">
        <string>0.8</string>
       </property>
       <property name="placeholderText">
        <string>0.8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="Form_alg_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="VInput1">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="VInput2">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="TcInput">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>111</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rBtn1">
       <property name="text">
        <string>T 0-10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtn2">
       <property name="text">
        <string>T 11-20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtn3">
       <property name="text">
        <string>T 21-Tc</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
